website/docs/r/mssql_elasticpool.html.markdown

```diff
@@ -63,6 +64,8 @@ The following arguments are supported:
 
 * `per_database_settings` - (Required) A `per_database_settings` block as defined below.
 
+* `max_size_bytes` - (Required) The max data size of the elastic pool.
```

Comment:
In the schema file this is listed as optional, however here it is listed as required?